app/code/Magento/Eav/Model/ResourceModel/Entity/Attribute/Collection.php

```diff
@@ -23,6 +23,13 @@ class Collection extends \Magento\Framework\Model\ResourceModel\Db\Collection\Ab
      */
     protected $_addSetInfoFlag = false;
 
+    /**
+     * All attribute sets filter is applied flag
+     *
+     * @var bool
+     */
+    protected $_setFilterApplied = false;
```

Comment:
Please remove the single underscore from property name (https://www.php-fig.org/psr/psr-2/#42-properties) also I propose to rename property to $filterApplied to correspond the value of the property (not reference to an action)